### DISCUSSION TITLE
Phishing Website Report: Unauthorized Use of Metlife's Branding

### DISCUSSION THREAD

**Discussion by anastasiiasavchynpf** [vercel/community#7445] [Help] [answered]
### Summary

**Client Affected:** Metlife  
**Malicious Website:** hxxp://pnbmetlife(.)vercel(.)app/  
**Original Website:** [metlife.com](https://www.metlife.com)

### Summary

The website at `hxxp://pnbmetlife(.)vercel(.)app/` is conducting a phishing attack by impersonating Metlife. It uses Metlife's name and logo without permission, misleading users into believing it is an official Metlife site.

### Details

**Original Website:**
- **URL:** [https://www.metlife.com](https://www.metlife.com)
- **Description:** Metlife is a global insurance and benefits provider.

**Malicious Website:**
- **URL:** hxxp://pnbmetlife(.)vercel(.)app/
- **Description:** The site falsely presents itself as an official Metlife webpage, using unauthorized branding to deceive users.

**Impact:**
- **User Risk:** Potential theft of personal information.
- **Brand Damage:** Erodes customer trust and harms Metlife’s reputation.
- **Legal Issues:** Violation of intellectual property rights.

### Recommendations
- **Immediate Takedown:** Contact the hosting provider, Vercel, to request the removal of the malicious website.
- **User Alert:** Inform users about the phishing site to prevent data compromise.

**Comment by amyegan** [comment, marked as answer]
Thanks for letting us know about this potential phishing site. If you haven't already, please send your report to [abuse@vercel.com](mailto:abuse@vercel.com) to generate a ticket. That way it will be tracked correctly in our system and you'll get updated about it.

I and the other members of this community do not have access to the systems and tools needed to handle phishing reports, but your email report will get you in touch with the specialists who can investigate and address these issues.

[a comment by an automated account is not shown]
